src/main/groovy/edu/oregonstate/mist/osuevents/db/EventsDAO.groovy

```diff
@@ -44,10 +44,13 @@ public interface EventsDAO extends Closeable {
             OWNER
         FROM EVENTS_EVENTS
         WHERE (EVENT_ID = :eventID OR :eventID IS NULL)
+	 AND (CREATED_AT >= SYSDATE - :changedInPastHours/24 OR :changedInPastHours IS NULL)
+	 AND (UPDATED_AT >= SYSDATE - :changedInPastHours/24)
```

Comment:
Should you add a `OR :changedInPastHours IS NULL` in this line too?

Reply:
Yes, this was the source of the bugs mentioned.